core/services/validation.py

```
@@ -43,14 +43,17 @@ class HeaderHashCheck(ValidationCheck):
     """
     name = "header_hash_check"
     
-    def _execute(self, message: str, allowed_prefix: str, hash_length: int,
-                 **kwargs) -> Tuple[int, Optional[str]]:
+    def _execute(self, **kwargs: Any) -> Tuple[int, Optional[str]]:
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)